client/clinic-react-client/src/components/offices/ActionPages/CreateWindow.tsx

```diff
@@ -0,0 +1,54 @@
+import { Button, FormControl, Input, InputLabel, MenuItem, Select } from '@mui/material';
+import cl from './CreateWindow.module.css'
+import { useState } from 'react';
+
+const CreateWindow: React.FC<CreateParams> = (props) => {
+
+    const [address, setAddress] = useState('');
+    const [phone, setPhone] = useState('');
```

Comment:
You can use single useState for the all 3 states.
{
address,
phone,
active
}